pkg/controller/smtpcredentialset/smtpcredentialset_controller_test.go

```diff
@@ -107,6 +107,9 @@ func TestReconcileSMTPCredentialSet_Reconcile(t *testing.T) {
 								},
 							}, "", nil
 						},
+						GetReconcileTimeFunc: func(smtpCreds *v1alpha1.SMTPCredentialSet) time.Duration {
+							return time.Second * 10
```

Comment:
Much as I like the readability of `* 10` I think we should add the numbers as variables

Reply:
good point, will do

Reply:
going to address this and other refactoring in a separate pr. merging to unblock e2e tests